.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: c
+sudo: required
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+    - os: linux
+      dist: xenial
```

Comment:
Right, until your comment I thought that would work. I requested to change travis-ci behaviour to fail if it encounters unsupported dists at https://github.com/travis-ci/travis-ci/issues/7296.